examples/federated_statistics/df_stats/README.md

```diff
@@ -0,0 +1,314 @@
+We split the training data and test data into two clients (one client with training, and another client with test dataset) 
+
+```applicaiton.conf
+
+fed_stats {
+data {
+features = ["Age", "Workclass", "fnlwgt", "Education", "Education-Num", "Marital Status",
+             "Occupation", "Relationship", "Race", "Sex", "Capital Gain", "Capital Loss",
+            "Hours per week", "Country", "Target"]
+
+        clients { # inherit common properties
+            site-1 {
+                url = "https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data"
+                filename = "data.csv"
+                skiprows = [] # Line numbers to skip (0-indexed) or number of lines to skip (int) at the start of the file.
+            }
+            site-2 {
+                url = "https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.test"
+                filename = "data.csv"
+                skiprows = [0] # Line numbers to skip (0-indexed) or number of lines to skip (int) at the start of the file.
+            }
+        }
+    }
+
+}
+
+```
+The application.conf format is [pyhocon](https://github.com/chimpler/pyhocon) format.
+
+Now we use data utility to download UCI datasets to separate client package directory in POC workspace
+
+<poc workspace>/site-1/data.csv
+<poc workspace>/site-2/data.csv
+
+
+```
+python3 data_utils.py  -h
+
+usage: data_utils.py [-h] [--prepare-data]
+
+fed_stats parser
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --prepare-data        prepare data based on configuration
+
+```
+
+```
+python3 data_utils.py  --prepare-data
+
+prepare data for poc workspace:/tmp/nvflare/poc
+remove existing data at /tmp/nvflare/poc/site-1/data.csv
+wget download to /tmp/nvflare/poc/site-1/data.csv
+100% [..........................................................................] 3974305 / 3974305remove existing data at /tmp/nvflare/poc/site-2/data.csv
+wget download to /tmp/nvflare/poc/site-2/data.csv
+100% [..........................................................................] 2003153 / 2003153done with prepare data
+
+```
+If your poc_workspace is in a different location, use the following command
+
+```
+export NVFLARE_POC_WORKSPACE=<new poc workspace location>
+```
+then repeat above
+
+
+## 3. Compute the local and global statistics for each numerical feature
+
+### 3.1 Specify client side configuration
+
+We are using a built-in NVFLARE executor,  
+
+```
+ "executor": {
+        "id": "Executor",
+        "path": "nvflare.app_common.executors.statistics_executor.StatisticsExecutor",
+        "args": {
+          "generator_id": "df_stats_generator",
+          "min_count" : 10,
+          "min_random": 0.1,
+          "max_random": 0.3
+  },
+
+```
+here we specify a minimum of 10 rows (min_count), if the number of rows less than min_count, the job will fail.
```

Comment:
This line is not clear to me, does this line saying we need 10 clients?
Or it means each site should have at least 10 records.